spi/identity-hub-spi/src/main/java/org/eclipse/dataspaceconnector/identityhub/model/credentials/VerifiableCredential.java

```diff
@@ -34,13 +35,19 @@ public class VerifiableCredential {
 
     private String id;
 
+    private Map<String, Object> credentialSubject;
+
     private VerifiableCredential() {
     }
 
     public String getId() {
         return id;
     }
 
+    public Map<String, Object> getCredentialSubject() {
+        return credentialSubject;
```

Comment:
does this violate immutability?

Reply:
When creating it from the builder method, Map.copyOf is used. According to the javadoc `Map.copyOf` returns an unmodifiable Map. So it is immutable.